components/ProposalStateBadge.tsx

```diff
@@ -46,6 +48,8 @@ function getBorderColor(proposalState: ProposalState, otherState: OtherState) {
         : 'border-[#5DC9EB]'
     case ProposalState.Voting:
       return otherState.votingEnded ? 'border-[#5DC9EB]' : 'border-[#8EFFDD]'
+    default:
```

Comment:
if proposalStates ever change again in the future then this will error and remind us to be exhaustive.